tasks/tests/test_base.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+""" This file tests the classes and functions in tasks/base.py """
+
+from requests import Response
+from tasks.base import check_response
```

Comment:
almost perfect! There would usually be an empty line in between these two import groups